app/Notifications/StudentDigestSender.php

```diff
@@ -25,19 +25,14 @@ public function __construct(Mailer $mailer)
     public function sendNotifications(array $notifications, Student $user): void
     {
 
-        // Disable function usableNotifications because it fails. Not sure why it is needed!
```

Comment:
Door niet te checken of het het goeie type notification is, is er een kans dat je een verkeerde type notificatie gaat groeperen in een `*Digest` email.

Hier worden namelijk de notificaties zonder te filteren doorgegeven: https://github.com/HUInstituteForICT/workplacelearning/blob/d33036418a804f77284de3cfe210f078bc3fd204/app/Notifications/NotificationRouter.php#L35-L43

Als er dus een `FolderFeedbackGiven` notificatie en een `InactiveStudent` notificatie klaarstaan, zal er een digest mail worden verstuurd met beide notificaties, en een inactive student mail met beide notificaties daarin gerenderd.

Zie https://github.com/HUInstituteForICT/workplacelearning/blob/d33036418a804f77284de3cfe210f078bc3fd204/app/Mail/InactiveStudent.php#L32-L36